src/components/Timer/Timer.js

```diff
@@ -1,9 +1,221 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import { buildStyles, CircularProgressbar } from "react-circular-progressbar";
+import "react-circular-progressbar/dist/styles.css";
+import { toast } from "react-toastify";
+import "./Timer.css";
+import { AiFillPauseCircle, AiFillPlayCircle } from "react-icons/ai";
+import { BiReset } from "react-icons/bi";
+
+const Timer = ({ totalDuration, longBreak, shortBreak, setTaskCompleted }) => {
+  const [minutes, setMinutes] = useState(totalDuration);
```

Comment:
All logically related states can be grouped into one state object. This will improve code readability